docs/TpchBenchmark.md

```diff
@@ -0,0 +1,453 @@
+    <tr><td>num_splits_per_file</td><td>Row Group Size of Data</td><td>Row Group Size of Data</td></tr>
+    <tr><td>max_coalesce_bytes</td><td>Minimum of 90MB</td><td>Minimum of 90MB</td></tr>
+    <tr>
+        <td>max_coalesce_distance_bytes</td>
+        <td>Workload dependent<super>**</super></td>
+        <td>Workload dependent<super>**</super></td>
+    </tr>
+    <tr><td>parquet_prefetch_rowgroups</td><td>Best is 1</td><td>Best is 1</td></tr>
+    <tr><td>split_preload_per_driver</td><td>Best is 2</td><td>Best is 2</td></tr>
+    <tr><td>cache_prefetch_min_pct</td><td>Best is 80</td><td>Best is 80</td></tr>
+    <tr>
+        <td colspan="3" style="font-size: 8pt">
+&nbsp;&nbsp;*&nbsp;<b>vCPUs</b> = (cores * hyper-threads)<br />
+**&nbsp;Wide tables and few columns retrieved per row can lead to many I/O requests, suggest increasing this value based on testing and the need to reduce small I/O requests.
+        </td>
+    </tr>
+</table>
+
+## Optimizations for the TpchBenchmark (Single Process Use Case)
+
+### ___num_drivers___
+This configuration option describes the number of resources available to process query plan tasks. This can greatly improve performance of CPU bound workloads. The recommendation in the table above seems to be optimal for TPCH-like workloads.
+
+### ___num_io_threads___
+This configuration option describes the number of threads available per driver to retrieve data from the network source. If the workload is I/O bound, then increasing this number is beneficial if the data is fewer requests of larger chunks as opposed to many smaller requests.
+
+### ___cache_gb___
+This configuration option is useful for workloads that read the same data several times per query but only applies to the single process use case. NOTE: There is a SSD Caching option in Velox but it to is ONLY useful in the single process use case.
+
+### ___num_splits_per_file___
+This configuration option is best when the data set count of row groups matches this value. The affect in overall performance appears based on testing to be small, however.
+
+## Optimizations for All Workloads (Both Use Cases)
+
+### ___max_coalesce_bytes___
+This configuration option is the maximum bytes coalesced into a single request to the data source. This was tested from the default 128MB to 2GB, and the overall improvement was small as size increased. Capturing request data did show larger and fewer requests but not enough to vastly improve I/O performance. 
+
+### ___max_coalesce_distance_bytes___
+This configuration option is the maximum byte distance between needed data in the same file at the data source that can be coalesced. Increasing this value would theoretically reduce the number of requests and increase each request size. However, if made too large the query will return too many un-needed bytes and could decrease I/O performance. This plus __max_coalesce_bytes__ should be fine-tuned for the workload being run.
+
```

Comment:
Should `parquet_prefetch_rowgroups` `split_preload_per_driver` and `cache_prefetch_min_pct` be added to a workload dependent part?

Reply:
So I found that these did not have a noticeable impact in the tool's performance. That is not to say that a different workload  could not benefit but I have no data on this so I cannot make any claims.

Reply:
Got it, thanks a lot for your explainition!